train.py

```diff
@@ -136,7 +137,7 @@ def initialize_trainer(epochs, patience, model_name, l_rate_enc, l_rate_cl, weig
                          gpus=1 if torch.cuda.is_available() else 0,
                          max_epochs=epochs,
                          callbacks=[early_stop_callback],
-                         progress_bar_refresh_rate=1)
+                         progress_bar_refresh_rate=0)
```

Comment:
Is this because you don't like the display of the progress bar? Or is it because the job output files geet so long?

Reply:
Yea it would just get long and I was annoyed by it since it's not like I can see a progress bar when I run the job